roles/splunk_search_head/tasks/setup_multisite.yml

```diff
@@ -12,7 +12,9 @@
   become: yes
   become_user: "{{ splunk.user }}"
   register: set_new_master
-  until: set_new_master.rc == 0
+  until: set_new_master.rc == 0 or (set_new_master.rc != 0 and "Cannot edit this searchhead. Use 'splunk edit cluster-master' to edit information for this searchhead." in set_new_master.stderr)
+  changed_when: set_new_master.rc == 0
```

Comment:
Looks like the linting caught that `changed_when` is a duplicate key here. Can we remove one of these?

Reply:
Thank you for the quick review @nwang92 and sorry about, I corrected this in the PR and found the lint tests to run in local.